spec/mailers/user_mailer_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe UserMailer, type: :mailer do
+  let(:user) { create(:user) }
+  before(:each) do
+    stub_const("NOTIFY_WELCOME_EMAIL_TEMPLATE", "123")
```

Comment:
FYI `ClimateControl` is available if you need to stub out environment variables.

Reply:
Thanks for reminding me, using that will make this more consistent.